docker/Dockerfile

```diff
@@ -2,15 +2,20 @@ ARG BRANCH=${BRANCH:-ce}
 ARG OPTIMIZER_ORTOOLS_VERSION=${OPTIMIZER_ORTOOLS_VERSION}
 ARG REGISTRY=${REGISTRY:-registry.mapotempo.com/}
 ARG VROOM_VERSION=${VROOM_VERSION}
+ARG WITHOUT="development test"
 
 # Install Vroom
 FROM vroomvrp/vroom-docker:${VROOM_VERSION:-v1.8.0} as vroom
 
 # Rake
 FROM ${REGISTRY}mapotempo-${BRANCH}/optimizer-ortools:${OPTIMIZER_ORTOOLS_VERSION:-v1.4.0}
+ARG WITHOUT
```

Comment:
This might be a Dockerfile specific thing but in case it is a typo, is this double definition normal? `WITHOUT` is already defined as argument on line 5 above.

Reply:
Yes it's definitely a Dockerfile specific trick. `ARG` must be present in second `FROM` block to be used in.

Reply:
Cheers! Then I think this is ready.